acceptance_tests/features/environment.py

```diff
@@ -30,8 +33,11 @@
 
 timings = {}
 
-
 fixture_scenario_registry = {
+    'fixture.setup.prepare.data.for.new.survey.with.internal.user':
```

Comment:
rename as @fixture.setup.survey.data.with.internal.user

Reply:
Disagree with this one , the reason is that this fixture does not create a survey that is done through @fixture.setup.data.survey.with.internal.user. What this one does is prepare the data on the context , which I why I went for a long winded name . Don't mind changing the name but it needs to reflect that it does not create a survey

Reply:
reason being all others implicitly create survey, collection exercise etc... maybe @fixture.setup.survey.metadata.with.internal.user is more accurate?